exhibitor-appliance.yaml

```diff
@@ -1,14 +1,21 @@
 SenzaInfo:
   StackName: exhibitor
   Parameters:
-    - DockerImage:
-        Description: Docker image of exhibitor
+    - DockerVersion:
```

Comment:
What about using `DockerTag` to follow the official docker nomenclature (https://docs.docker.com/engine/reference/glossary/#tag) ?